build.gradle

```diff
@@ -0,0 +1,63 @@
+group 'org.bitbucket.lykke'
```

Comment:
why bitbucket - can we create or use org.github ?

Reply:
this group id is used for out kotlin utils lib and we decided in our chat to use it also for our other libraries